components/dataentry/dataentry-controller.js

```diff
@@ -2,6 +2,10 @@
 
 import {disableCompleteIncompleteEventConfirmation} from "../../ks_patches/custom_override_flags";
 import { INNREISEINFORMASJON_PROGRAM_STAGE_ID} from "../../utils/constants";
+import {
+    INDEKS_SISTE_ISOLASJONSDATO,
+    NÆRKONTAKT_SISTE_KARANTENEDATO

```

Comment:
liker man æ i variabelnavn? mener at vi gikk på en smell der for en tid tilbake?

Reply:
Det ser ut som det fungerer helt særlig på noen tidligere konstanter som lå der, men jeg kjenner på den samme frykten og vi kan gjærne endre.